pkg/options/diffoptions.go

```diff
@@ -47,17 +49,24 @@ func (o *DiffOptions) Run(cmd *cobra.Command, args []string) error {
 	if err != nil {
```

Comment:
just saw you put the Run in the "options" package, please move it to be with the cmd.

the options should be "data model" only.